app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -215,6 +214,14 @@ const ContentForm: React.FC<IContentFormProps> = ({
     }
   }, [id, fetchContent]);
 
+  const goToNext = React.useCallback(
```

Comment:
A separate button to move to next.  This button does not save however.  I'd like to figure out how to determine whether it should save first.